vncviewer/DesktopWindow.cxx

```diff
@@ -467,7 +467,32 @@ void DesktopWindow::draw()
     // primary screen rather than the entire window
     if (fullscreen_active() && fullScreenAllMonitors) {
```

Comment:
Can't this be simplified further now by just checking `fullscreen_active()`?

Reply:
Definitely. Fixed.